frontend/src/pages/user_home.tsx

```diff
@@ -52,7 +56,30 @@ export default function HomePage() {
             </li>
           );
         })}
+
+        <li>
+          <Link
+            to={`/new`}
+            className={clsx(
+              'block',
+              'p-4',
+              'aspect-square',
+              'bg-white',
+              'font-medium',
+              'rounded-lg',
+              'border',
+              'border-gray-200',
+              'hover:shadow-lg',
+              'hover:scale-105',
+              'transition-all',
+              'flex',
+            )}
+          >
+            {' '}
+            <AiOutlinePlus size="1.75rem" />
```

Comment:
I think the aria label (or `visually-hidden` span) is missing here for accessibility.